### PR TITLE
Add Telepen/Telepen Numeric

### PR DESCRIPTION
**Pull request by illuminatusds**
Add support for Telepen / Telepen numeric codes

### CODE REVIEW

File: src/Types/TypeTelepen.php [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+namespace Picqer\Barcode\Types;
+
+use Picqer\Barcode\Barcode;
+use Picqer\Barcode\BarcodeBar;
+use Picqer\Barcode\Exceptions\InvalidFormatException;
+
+define('TELEPEN_START_CHAR', '_');
+define('TELEPEN_STOP_CHAR',  'z');
+define('TELEPEN_ALPHA',   'alpha');
+define('TELEPEN_NUMERIC', 'numeric');
```
Comment (casperbakker) [inline src/Types/TypeTelepen.php]: I don't like these broad define's in de codebase. I think these can be private constants in the class itself, so it is encapsuled from the rest of the application.

File: src/Types/TypeTelepen.php [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+        $dest .= $this->telepen_lookup_table[$check_digit];
+        $dest .= $this->telepen_lookup_table[ord(TELEPEN_STOP_CHAR)]; // Stop
+
+        return $dest;
+    }
+
+    private function createTelepenConversionTable()
+    {
+        $this->telepen_lookup_table = [
```
Comment (casperbakker) [inline src/Types/TypeTelepen.php:165]: Can this be formatted better? Can you also provide a comment about how this lookup table works?

**Review by casperbakker — COMMENTED** [review]
I would be happy to add Telepen to this library. Could you look at my comments and improve the code a bit? Then I can add it to the library.

File: src/Types/TypeTelepen.php (hunk 1 above)
Comment (illuminatusds) [reply src/Types/TypeTelepen.php]: changed to `const`

File: src/Types/TypeTelepen.php (hunk 2 above)
Comment (illuminatusds) [reply src/Types/TypeTelepen.php:165]: Small amount of explanation added to table creation, and grouped items into consistent rows of 4 items

**Comment by casperbakker** [comment]
Thanks!

**Pull request merged**